cypress/integration/ambianic-tests/settings.spec.js

```diff
@@ -2,8 +2,8 @@
 
 context('Settings', () => {
     before(() => {
-      cy.visit('http://localhost:8080')
-      cy.get('[data-cy=settings]').click()
+      cy.visit('http://localhost:8080/settings')
```

Comment:
@vickywane I see the click() action has been removed. Is this test useful without any assertion?

Reply:
The setting button no longer exists on the home page due to the new installation flow. 

The purpose of the click action was to navigate to the settings page which using `visit` does. 

@ivelin this also applies to similar test suites that had to do with the previous Home screen.

Reply:
OK, makes sense. Closing.

Reply:
@vickywane actually, I agree with the change to visit /settings, but we still need to make some sort of reasonable assumption about the contents of the settings page.

Reply:
I think the code coverage of the setting page is out of the scope of this PR.